### ISSUE TITLE
Adding a new message?

### ISSUE THREAD

**Issue by hendrixgr** [paparazzi/pprzlink#132]
Hi
I am not sure if this is an issue but i would love to add a new message about my airborne Yagi antenna

    <message name="AIRBORNE_ANTENNA" id="181">
      <field name="mag_heading_deg"   type="float"/>
      <field name="gps_heading_deg"  type="float"/>
      <field name="ant_azimuth_deg" type="float"/>
      <field name="ant_elevation_deg" type="float"/>
    </message>

The message can be different i just used the above code as an example.
Basically i can't find a message to transmit the mag Heading in degrees without interfering with other modules plus that there is no message for the airborne antenna.
Chris

**Comment by OpenUAS** [comment]
You can add you own messages.xml for the time being and add it, copy your adjusted messages.xml in your ~/paparazzi/conf/ directory then a `make clean && make` and start using it. 

Note that there was already a message for this purpose in the past as I recall, maybe I'm wrong, but search current code first before you need to add something.
